hist/hist/test/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ ROOT_ADD_GTEST(testMapCppName test_MapCppName.cxx LIBRARIES Hist Gpad)
 ROOT_ADD_GTEST(testTGraphSorting test_TGraph_sorting.cxx LIBRARIES Hist)
 ROOT_ADD_GTEST(testSpline test_spline.cxx LIBRARIES Hist)
 ROOT_ADD_GTEST(testTF1Simple test_tf1_simple.cxx LIBRARIES Hist RIO)
+ROOT_ADD_GTEST(testTFormulaPolN test_TFormulaPol.cxx LIBRARIES Core Hist)
```

Comment:
I would suggest to add all your tests directly in `hist/hist/test/test_TFormula.cxx`